src/Security/Security/ChangeLog.md

```diff
@@ -18,9 +18,9 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-* Added new cmdlet: 'Get-AzSecurityTopology'
+* Added new cmdlet: 'Get-AzAllowedConnections'
```

Comment:
please insert this release note under
## Upcoming Release
section and add the previous notes back
* Added new cmdlet: 'Get-AzAllowedConnections'
* Added new cmdlet: 'Get-AzSecurityTopology'